cfgov/unprocessed/css/enhancements/layout.less

```diff
@@ -321,6 +321,49 @@
 }
 
 
+// Center content and bleed full-width images into gutters
+// Used on our story pages
+.content__center-image-bleed {
+
+    & > section {
+        margin: 0 auto;
+        max-width: 41.875rem;
+
+        .o-full-width-text-group {
+            overflow: inherit;
+        }
+
+        img.bleed-image {
```

Comment:
We should probably settle on `bleed-image` or `image-bleed` (see `content__center-image-bleed` above)

Reply:
Do we need `img` here? Maybe just have it `.bleed-image` (if we switched to `<picture>` or something down the road.

Reply:
Changed everything to `.image-bleed` in https://github.com/cfpb/consumerfinance.gov/commit/e9f1d87096db0a1e546dadc7b3f2dc17c0bd8d2f 👍